.github/workflows/publish.yml

```diff
@@ -0,0 +1,72 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#publishing-to-package-registries
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Upload Python Package
+
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: read
+
+jobs:
+#  TODO issue #11: deduplicate this, and trigger the test.yml directly
```

Comment:
Not very happy with duplicating test.yml here, but I couldn't get it to work and did not want to spend more time on this, so created a separate issue